labelbox_dev/data_row.py

```diff
@@ -32,6 +32,13 @@ class CreateDataRowType(TypedDict):
     media_type: Optional[str]
 
 
+class UpdateDataRowsWithIdType(TypedDict):
+    id: str
```

Comment:
Is the data_row to update always selected by `id`? I wonder what the workflow would look like if the user wants to update `row_data` using a `global_key`, do they fetch data_row_ids by global_keys first?

Reply:
Good callout. We should enable it for global_key as well

Reply:
https://labelbox.atlassian.net/browse/AL-4973